crates/ide/src/status.rs

```diff
@@ -34,15 +35,18 @@ fn macro_syntax_tree_stats(db: &RootDatabase) -> SyntaxTreeStats {
 // image::https://user-images.githubusercontent.com/48062697/113065584-05f34500-91b1-11eb-98cc-5c196f76be7f.gif[]
 pub(crate) fn status(db: &RootDatabase, file_id: Option<FileId>) -> String {
     let mut buf = String::new();
+    let count = profile::countme::get_all();
```

Comment:
Let's move this inside the `if`.

Reply:
Got it.

Reply:
[`4d8fe62` (#9269)](https://github.com/rust-analyzer/rust-analyzer/pull/9269/commits/4d8fe6208eb5543183bc8668ca4fefef50a67bf9)